src/pages/home/components/introCardproduct/introCard.tsx

```diff
@@ -0,0 +1,55 @@
+import { useSelector } from 'react-redux';
+import styles from './introCard.module.scss';
+import { RootState } from '../../../../redux/store';
+import { useNavigate } from 'react-router-dom';
+import routes from '../../../../utils/routes';
+import useFetch from '../../../../hooks/useFetch';
+import RatingsCounter from '../../../../designComponents/ratingsCounter/ratingsCounter';
+
+interface SingleProduct {
+  id: number;
+  title: string;
+  price: number;
+  rating: {
+    rate: number;
+    count: number;
+  };
+  image: string;
+  category: string;
+  description: string;
+}
+
+export default function IntroCrads() {
+  const navigate = useNavigate();
+  const token = useSelector((state: RootState) => state.token.value);
+  const handleRedirectUser = () => (token ? navigate(routes.addToCart) : navigate(routes.login));
+  const { isLoading, data, error } = useFetch('https://fakestoreapi.com/products');
+  const newData: SingleProduct[] | null = data as SingleProduct[] | null;
```

Comment:
You can use Data from backend, not dummy data

Reply:
Also the Recommended product cards needs to be changed. you can refer to Figma design.
![Screenshot 2023-06-10 180022](https://github.com/atlp-rwanda/Team-Sostene-E-commerce-fe/assets/93517096/2b81eb68-d2b1-461d-9863-07c9ab6c515c)